build-rpi-raspbian/webif.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+# start:     	starts a minimalistic web server
+# 	    	shows the status of OpenHABianPi installation
+# restart:   	checks if webserver is running 
+# inst_done: 	create finish message and link to http://$hostname:8080
+# cleanup:	stops the webserver
+#	     	removes all no longer needed files
+
+port=80 # Port the webserver is listing to
+source /etc/openhabian.conf # to get the hostname
+
+if [ $1 = "start" ]; then
+  mkdir /tmp/webif
+  ln -s /boot/first-boot.log /tmp/webif/first-boot.txt
+  echo "<html>
+        <head>
+        <title>openHABian</title>
+        <meta http-equiv="refresh" content="5" />
+        <meta http-equiv="Cache-Control" content="no-cache, no-store, must-revalidate" />
+        <meta http-equiv="Pragma" content="no-cache" />
+        <meta http-equiv="Expires" content="0" />
+        </head>" > /tmp/webif/index.html
+  echo "<body>
+        <h1>openHABian Installation Status</h1>
+        the log will be refreshed automatically every 10 seconds
```

Comment:
My question regarding 10 seconds was aimed at `http-equiv="refresh" content="5"`

Reply:
a leftover from my tests, this line will be removed in the next commit